skin.tcl

```diff
@@ -6,6 +6,8 @@ source "[homedir]/skins/default/standard_includes.tcl"
 
 set ::skindebug 0
 set ::debugging 0
+set ::profile_to_restore_after_cleanup {}
```

Comment:
Maybe this should be moved into the settings? Else I would be lost on app restart?

Reply:
Good question. From my point of view this is not a setting, but another concept. It's temporary state during a workflow (the cleanup). Therefore I personally would not save it as a setting (from a micro architectural point of view). I think the clean solution (haha play on words) would be to provide a general way to persist state that should survive app restart. Another example for this state could be the graph of the last shot run. But I see such a mechanism a bit out of scope for that small feature -- and would not introduce it by PR without discussing. Personally I don't think it's a problem if a user selects clean, restarts the app, runs clean and afterwards the clean profile stays selected. But if have another perspective on this, I'll be happy to add it to settings.

Reply:
Then please move it to the settings, and I will add a persistent state array soon-ish and discuss the options with John on how we might benefit appwide

Reply:
Just pushed the change you asked for (I'll only save the settings array to file if the option is enabled since it makes no sense otherwise).